src/main.rs

```diff
@@ -1,3 +1,10 @@
-fn main() {
-    println!("Hello, world!");
+use actix_web::{App, HttpServer};
+use rust_sns::router;
+
+#[actix_rt::main]
+async fn main() -> std::io::Result<()> {
+    HttpServer::new(|| App::new().configure(router::router))
+        .bind("localhost:8000")?
```

Comment:
[IMO]
細かいですがポートは分けて書いたほうが読みやすくなります。
```suggestion
        .bind(("localhost", 8000))?
```